argus/backend/controller/testrun_api.py

```diff
@@ -165,12 +166,15 @@ def set_testrun_assignee(test_id: str, run_id: str):
     }
 
 
-@bp.route("/test/<string:test_id>/run/<string:run_id>/issues/submit", methods=["POST"])
+@bp.route("/test/<string:test_id>/run/<string:run_id>/issues/<string:issue_type>/submit", methods=["POST"])
 @api_login_required
-def issues_submit(test_id: str, run_id: str):
+def issues_submit(test_id: str, run_id: str, issue_type: str):
```

Comment:
should be str literal, so people know what to pass here

Reply:
or even better, detect from `payload["issue_url"]`

Reply:
Removed and done